### PR TITLE
[Shapes] Implicitly animate shape path on resizing

### PR DESCRIPTION
**Pull request by samnm**

### CODE REVIEW

File: components/ShadowLayer/src/MDCShadowLayer.h [hunk 1]
```diff
@@ -77,3 +77,10 @@
 @property(nonatomic, getter=isShadowMaskEnabled, assign) BOOL shadowMaskEnabled;
 
 @end
+
+/**
+ Subclasses can depend on MDCShadowLayer implementing CALayerDelegate actionForLayer:forKey: in
```
Comment (romoore) [inline components/ShadowLayer/src/MDCShadowLayer.h:82]: Does MDCShadowLayer need to explicitly implement `actionForLayer:forKey:` or does CALayer take care of that?

**Review by romoore — APPROVED** [review]

**Comment by romoore** [comment]
Is it possible to backlog an Issue to write a couple of unit tests for the expected behavior?

File: components/ShadowLayer/src/MDCShadowLayer.h (hunk 1 above)
Comment (samnm) [reply components/ShadowLayer/src/MDCShadowLayer.h:82]: As far as I know there is no way to rely on the built-in CAAction creation for headless layers so we explicitly implement `actionForLayer:forKey:`.

Comment (romoore) [reply components/ShadowLayer/src/MDCShadowLayer.h:82]: Ack. I missed the method definition in my earlier search. :-S

**Pull request merged**